src/js/rx/reducers/prescriptions.js

```diff
@@ -1,11 +1,35 @@
 import set from 'lodash/fp/set';
+import _ from 'lodash';
 
 const initialState = { items: [] };
 
+function sortByName(obj) {
+ /*
+  Making all values the same case, to prevent
+  alphabetization from getting wonky.
+  */
+  return obj.attributes['prescription-name'].toLowerCase();
+}
+
+function sortByFacilityName(obj) {
+  return obj.attributes['facility-name'];
+}
+
+function sortByLastRequested(obj) {
+  return new Date(obj.attributes['ordered-date']).getTime();
```

Comment:
Per https://github.com/department-of-veterans-affairs/kudos-team/blob/master/Prescriptions/design/Rx%20statuses.md, it's supposed to be `refill-date`.